File: sort/Python3/sort.py
import sys

N = 1 << 22
print_out = False
if len(sys.argv) >= 2:
    print_out = True
v = [None] * N
seed = 3
for i in range(N):
    v[i] = seed
    seed *= 27487
    seed %= 30491
v.sort()
if print_out:
    print(v[279121])

File: sort/JavaScript/sort.js
var v = new Array();
var seed = 3;
for(var i = 0; i < (1 << 22); i++)
{
    v[i] = seed;
    seed *= 27487;
    seed %= 30491;
}
v.sort(function(a, b){return a - b;});
if(process.argv.length >= 3)
    console.log(v[279121]);
